### ISSUE TITLE
Right margin (~20px) on iOS Mail app fix

### ISSUE THREAD

**Issue by endofgeneric** [emailmonday/Cerberus#17]
Was experiencing a margin to the right of the viewport in iOS Mail. 

![iosmail-issue](https://cloud.githubusercontent.com/assets/10222506/6962071/0132f958-d929-11e4-8aed-face7d83dcd4.png)

Added

`min-width: 100%;`

to the body tag and that fixed the issue.

EDIT: It seems to be a mixture of two things. One being the min-width fix above. I also had to remove

`table-layout: fixed;`

from the Yahoo alignment fix in the styles. Tested in Yahoo and iOS and everything loks fine. Seems like the Yahoo alignment fix is now redundant, I just needed to center-align the main container <td> (line 97).

Also removed `table-layout: fixed;` from line 97 as it was causing a Gmail bug - right hand side of Gmail interface overlaying the email content:

![gmail-issue](https://cloud.githubusercontent.com/assets/10222506/6962463/ae24376e-d92c-11e4-94d3-e9fab9bf52c8.png)

All good in Yahoo, Gmail and iOS now.

**Comment by TedGoas** [comment]
@endofgeneric Thanks for the bug and descriptive workaround! I'll edit the base template so this is fixed.

**Comment by endofgeneric** [comment]
No worries bud, glad it all made sense!

## Michael Clayton // End of Generic

## Brand Identity // Print & Digital Design // Videography

endofgeneric.com
dribbble.com/endofgeneric
twitter.com/endofgeneric
+44 (0)7594 855 499

**Comment by TedGoas** [comment]
@endofgeneric Hi Michael, few notes from my attempts:
- I wasn't able to replicate the spacing bug you described above. However I _have_ fixed spacing issues in the past few months, so if you're working with an older version of Cerberus, it might not have this fix. [I ran the current template through Litmus](https://litmus.com/pub/46b6de1) and don't see anything that matches your screenshot.
- I removed the references to `table-layout: fixed;` as you described, and Yahoo! remains good. Thanks for the tip; less code is better!
- Both Cerberus templates didn't center in Yahoo! desktop. I fixed this bug in the process.
